hashintel/hash
# Generate property table rows using entity type schema

## 🌟 What is the purpose of this PR?

This PR changes property table's row generation logic. Now it generates row using entity type's schema instead of entity's properties object. Because properties object of an entity could be an empty object. Which causes property table to render no rows at all.

## 🔗 Related links

- [Asana task](https://app.asana.com/0/1203179076056209/1203433085114587/f) _(internal)_


## 🔍 What does this change?




- Update the logic for generating `X empty, Y total` string. Now it uses the table rows instead of looking at property objects.


## ❓ How to test this?

1.  Checkout the branch / view the deployment
1.  Open entity editor `/new` page to create a new entity
2. New entity should show all of the properties instead of an empty property table
